### ISSUE TITLE
TFLite produces incorrect output for tf.math.l2_normalize with default axis on shape [N, 1]

### ISSUE THREAD

**Issue by ALinrunrun** [tensorflow/tensorflow#118181]
### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

2.21.0

### Custom code

Yes

### OS platform and distribution

Linux Ubuntu 22.04

### Python version

3.11

### Current behavior?

TensorFlow Lite produces a different result from Keras for tf.math.l2_normalize on a tensor with shape [3, 1].

The Keras model applies tf.math.l2_normalize(v) to the input. Keras matches the NumPy reference using the full tensor norm, but the converted TFLite model returns [1.0, 1.0, 1.0].

In the reproducer below, the expected output is approximately [0.3801878, 0.5974380, 0.7060631], while TFLite returns all ones.

Expected behavior?

The converted TFLite model should match the Keras output for tf.math.l2_normalize with the same input shape and default axis behavior.

### Standalone code to reproduce the issue

```shell
import os
os.environ["CUDA_VISIBLE_DEVICES"] = "-1"

import numpy as np
import tensorflow as tf


class M(tf.keras.Model):
    def call(self, v):
        return tf.math.l2_normalize(v)


X = np.array([[7.0], [11.0], [13.0]], dtype=np.float32)

model = M()
keras_out = model(tf.constant(X)).numpy()
ref = X / np.linalg.norm(X)

converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()

interpreter = tf.lite.Interpreter(model_content=tflite_model)
input_idx = interpreter.get_input_details()[0]["index"]
interpreter.resize_tensor_input(input_idx, list(X.shape))
interpreter.allocate_tensors()
interpreter.set_tensor(input_idx, X)
interpreter.invoke()

tflite_out = interpreter.get_tensor(interpreter.get_output_details()[0]["index"])

print(f"ref={ref.ravel().tolist()}")
print(f"keras={keras_out.ravel().tolist()}")
print(f"tflite={tflite_out.ravel().tolist()}")

if np.allclose(keras_out, ref) and not np.allclose(tflite_out, keras_out):
    print("BUG REPRODUCED: TFLite l2_normalize output differs from Keras/reference")
else:
    print("not reproduced")
```

### Relevant log output

```shell
ref=[0.3801878094673157, 0.5974379777908325, 0.7060630917549133]
keras=[0.3801878094673157, 0.5974379777908325, 0.7060630321502686]
tflite=[1.0, 1.0, 1.0]
BUG REPRODUCED: TFLite l2_normalize output differs from Keras/reference
```

**Comment by aravindmurali22** [comment]
Hi @ALinrunrun ,

Thank you for taking the time to report this and for your patience.

TFLite has [transitioned](https://developers.googleblog.com/en/tensorflow-lite-is-now-litert/) to LiteRT, Google's primary runtime for on-device AI. All active development, feature updates, and performance enhancements have shifted  to LiteRT. We encourage you to try running your model with LiteRT. If you still encounter this issue, please open a new issue with the updated details in the [LiteRT Repository](https://github.com/google-ai-edge/LiteRT).

For more information and step-by-step guidance on updating your project, please refer to the [LiteRT Migration Guide](https://developers.google.com/edge/litert/migration).
Thank you again!
